python/src/tokenize_sentence.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+
+use vtext::tokenize::Tokenizer;
+use vtext::tokenize_sentence::*;
+
+use crate::tokenize::BaseTokenizer;
+
+/// __init__(self, word_bounds=True)
+///
+/// Unicode Segmentation tokenizer
+///
+/// This implementation is a thin wrapper around the
+/// `unicode-segmentation <https://github.com/unicode-rs/unicode-segmentation>`_
+/// crate.
+///
+/// References
+/// ----------
+/// - `Unicode® Standard Annex #29 <http://www.unicode.org/reports/tr29/>`_
+#[pyclass(extends=BaseTokenizer)]
+pub struct UnicodeSentenceTokenizer {
```

Comment:
I don't understand why this file appears in the Github diff. It was merged in the last PR, wasn't it? Maybe merging upstream/master in again would help?

Reply:
Thanks. I have rebased the branch to the upstream/master